### ISSUE TITLE
Can't browse clipboard history with up/down

### ISSUE THREAD

**Issue by morojgovany** [tinytacoteam/zazu-clipboard#18]
Hi !
I don't know if it's an bug or my conf but I can't browse the clipboard history with the up and down arrows keys since the update to version 0.51 of zazu if I'm not mistaken. I removed the plugin directory and redownloaded it but no changes. It works with the mouse or **if I type a letter I can browse with keys**.

conf file :
```
  {
  "hotkey": "alt+space",
  "theme": "tinytacoteam/zazu-light-theme",
  "disableAnalytics": "true",
  "displayOn": "detect",
  "plugins": [
    "tinytacoteam/zazu-calculator",
    "tinytacoteam/zazu-package-manager",
    "tinytacoteam/zazu-file-finder",
    "tinytacoteam/zazu-template",
    "tinytacoteam/zazu-system",
    {
      "name": "tinytacoteam/zazu-clipboard",
      "variables": {
          "ClipboardKey": "cmd+shift+v"
      }
    },
    {
      "name": "tinytacoteam/zazu-fallback",
      "variables": {
          "rootSearches": [
              "google"
          ]
      }
    },
    "tinytacoteam/zazu-snippets",
    "blainesch/zazu-js-beautify",
    "brpaz/zazu-placehold.it",
    "blakek/zazu-ip-address"
  ]
}
```
system :
``
Linux 3.16.0-4-amd64 #1 SMP Debian 3.16.39-1+deb8u2 (2017-03-07) x86_64 GNU/Linux  
$ cat /etc/debian_version   
8.7``  

No errors in the logs apparently, if I type a caracter and browse, I can see the index working

edit : browsing works if I type "clip "in zazu

Thanks !

**Comment by bayleedev** [comment]
Thank you for reporting! [:

**Comment by bayleedev** [comment]
This looks related to tinytacoteam/zazu#262
cc @amjith

**Comment by amjith** [comment]
When is it interfering with the clipboard results? 

I'm confused.

**Comment by bayleedev** [comment]
It doesn't interfere with the results, it interferes with scrolling up and down.

**Comment by Koolstr** [comment]
This is still an issue, for me. Is there any progress on addressing this?

**Comment by bantya** [comment]
Yes this is still an issue as of July 16th, 2017.

**Comment by luiscachog** [comment]
Still an issue, as of September 24th, 2017. Can't browse clipboard history with up/down keys.

**Comment by tomer-ben-david** [comment]
I see not resolved, is there any workaround to this until this is resolved?

**Comment by bayleedev** [comment]
Not currently.

**Comment by mohithg** [comment]
This is still an issue.

**Comment by calebmeyer** [comment]
I'll mention this also does not work on windows (10 enterprise, 17134)

**Comment by CrajyB** [comment]
Wow, this issue has been open for 2 years! ;)

I'm on a Mac and also face the same issue. Having the ability to navigate items in your clipboard without using the mouse is a huge feature!

From the activity it would seem this is not going to get fixed. Maybe time to drop zazu completely, which is sad as I really liked it.

**Comment by linonetwo** [comment]
I know what's going on!

If you type any character, for example `a`

<img width="623" alt="屏幕快照 2019-05-17 下午6 48 04" src="https://user-images.githubusercontent.com/3746270/57923339-9df52780-78d4-11e9-81a8-ced0f1f99995.png">

Then up-down button will work. So this is a but on Zazu side. I'm going to create an issue on it.

**Comment by linonetwo** [comment]
A fix is on the way, may appear in Zazu 0.6.0

**Comment by aubreypwd** [comment]
Note if you type `.` you can search without it filtering out things _I think._ But it's a workaround.

**Comment by linonetwo** [comment]
In Zazu, if the **input is empty** you can use the arrow key to browse input history. But this feature broke the clipboard history since they all trying to browse the history.

That's why typing a `.` can fix it as a workaround, because it disables the "browse input history" behavior.

**Comment by linonetwo** [comment]
Fixed in latest Zazu 0.6.0.
